README.md

```diff
@@ -347,7 +360,7 @@ Joplin is currently available in the languages below. If you would like to contr
 
 - [Download Poedit](https://poedit.net/), the translation editor, and install it.
 - [Download the file to be translated](https://raw.githubusercontent.com/laurent22/joplin/master/CliClient/locales/joplin.pot).
-- In Poedit, open this .pot file, go into the Catalog menu and click Configuration. Change "Country" and "Language" to your own country and language.
+- In Poedit, open this .pot file, go into the Catalogue menu and click Configuration. Change "Country" and "Language" to your own country and language.
```

Comment:
Did you check if it's actually called "Catalogue" in Poedit? (I've got the French version so can't check)